scripts/docker/interpreter/Dockerfile

```diff
@@ -13,22 +13,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM apache/zeppelin:0.8.0
+FROM apache/zeppelin:0.11.2-SNAPSHOT
 MAINTAINER Apache Software Foundation <dev@zeppelin.apache.org>
 
-ENV SPARK_VERSION=2.3.3
-ENV HADOOP_VERSION=2.7
+ENV SPARK_VERSION=3.5.1
+ENV HADOOP_VERSION=3
```

Comment:
is this Dockerfile supposed to be used by user? if so, we should use ARG instead of ENV.

additionally, it's better to change it to SPARK_NAME or something else because the official spark binary artifacts have the below suffix 
```
-hadoop2 (removed since 3.4.0)
-hadoop3
-without-hadoop
```
users are also allowed to `dev/make-distribution.sh --name custom` to create spark bin tarball with `-custom` suffix.

Reply:
Now that I think about it, I agree with your comment. It does seem more appropriate for users to control the Spark bin name.
I'd reflect your suggestion in a new commit.